File: api/src/main/java/feliz/ifrs/previT/controller/responses/WeatherResponse.java
package feliz.ifrs.previT.controller.responses;

import lombok.*;

@Builder
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class WeatherResponse {

    private double longitude;
    private double latitude;
    private String weather;
    private String description;
    private double temperature;
    private double feelsLike;
    private double temperatureMin;
    private double temperatureMax;
    private double pressure;
    private double humidity;
    private double windSpeed;
    private String name;
    private String country;

}

File: app/src/ts/interfaces/weather-response.ts
export interface IWeatherResponse {
  longitude: number;
  latitude: number;
  weather: string;
  description: string;
  temperature: number;
  feelsLike: number;
  temperatureMin: number;
  temperatureMax: number;
  pressure: number;
  humidity: number;
  windSpeed: number;
  name: string;
  country: string;
}
